datawig/simple_imputer.py

```diff
@@ -320,8 +325,22 @@ def fit(self,
         :param weight_decay: regularizer (default 0)
         :param batch_size (default 16)
         :param final_fc_hidden_units: list dimensions for FC layers after the final concatenation
-
+        :param calibrate: Control automatic model calibration
+        :param class_weights: Dictionary with labels as keys and weights as values.
+                              Weighs each instance's contribution to the likelihood based on the corresponding class.
+        :param instance_weights: List of weights for each instance in train_df.
         """
+
+        # add weights to training data if provided
+        if class_weights is not None:
+            assert instance_weights is None
+            train_df = add_class_weight_to_df(train_df, class_weights, self.output_column, in_place=False)
+
+        elif instance_weights is not None:
+            assert class_weights is None
+            train_df = train_df.copy()
```

Comment:
minor comment: maybe it would be nicer to wrap the input data with an iterator that adds the instance weights on the fly instead of forcing a full copy of the input data.